app/config/index.js

```diff
@@ -40,6 +44,10 @@ value.caseSubscription = mqConfig.caseSubscription
 value.paymentTopic = mqConfig.paymentTopic
 value.validateTopic = mqConfig.validateTopic
 value.validationResponseSubscription = mqConfig.validationResponseSubscription
+value.createCrmCaseTopic = mqConfig.createCrmCaseTopic
+value.createCrmCaseSubscription = mqConfig.createCrmCaseSubscription
```

Comment:
Don't need to know the subscription for case create as we're publishing to that topic, not subscribing to it so don't need to know or care about subscriber details.